### ISSUE TITLE
dist/color-thief.js is not usable when targeting ES5, but color-thief.min.js works...

### ISSUE THREAD

**Issue by RudeySH** [lokesh/color-thief#166]
Currently, there's four files in the `dist` folder:
1. color-thief.js
2. color-thief.min.js
3. color-thief.mjs
4. color-thief.umd.js

To my confusion, `color-thief.min.js` does not appear to be the minified version of `color-thief.js`. I noticed the minified file declares `CanvasImage` and `ColorThief`, while the non-minified does not. Instead, the non-minified file contains some functions and uses (amongst other things) `const` and `require`, making it unusable when targeting ES5.

Maybe it's just me but I expected the non-minified file and the minified file to contain the same JS. Was this a mistake or is the non-minified version of `color-thief.min.js` simply not included with the npm package?

**Comment by lokesh** [comment]
@RudeySH I made some recent changes to the build process and dist files. They should be backwards compatible with older versions but the naming is a little unintuitive for the `.min` file unfortunately. Here is the list of all the files in the /dist folder and what formats they support:

- color-thief.js - CommonJS module for use in Node.
- color-thief.mjs - ES6 module. For modern browsers as well as Webpack and Rollup.
- color-thief.umd.js - UMD module. For simple script tag loading that exposes a global variable or for RequireJS AMD support.
- color-thief.min.js - Duplicate of color-thief.umd.js. Kept around to maintain backwards compatibility.

I've updated the demo page with info on this just recently: https://lokeshdhakar.com/projects/color-thief/#getting-started

Let me know if there is anything else I can help with. Reopen the issue if needed.
